python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -737,7 +738,7 @@ def _getChebyshevPhotoCalib(self, coefficients, err, xyMax, offset, scaling):
         # Take the zeropoint, apply the absolute relative calibration offset,
         # and whatever flat-field scaling was applied
         pars[:, :] = (coefficients.reshape(orderPlus1, orderPlus1) *
-                      10.**(offset / (-2.5)) * scaling)
+                      (offset*units.ABmag).to_value(units.nJy) * scaling)
```

Comment:
The offset here is delta-mag, not ABmag.  So an offset of say delta_mag = 0.1 should be applying a factor of 0.91 (original code), and now will apply an offset of 3311311214825.901.
Basically, the key is that the code will match whatever the units of the reference catalog are.

Reply:
Okay, I finally figured out for myself what's going on here, and it's correct, but it works because the units of the original returned coefficients from `fgcm`.  I'll have to document that better on the `fgcm` side, or change that.

Reply:
Methods like this are perfect use cases for simple unittests. I can help you write one, if you want.

Reply:
I guess this is DM-16525